stumpy/stump.py

```diff
@@ -16,7 +16,8 @@
 
 @njit(
     # "(f8[:], f8[:], i8, f8[:], f8[:], f8[:], f8[:], f8[:], f8[:], f8[:], f8[:],"
-    # "b1[:], b1[:], b1[:], b1[:], i8[:], i8, i8, i8, f8[:, :, :], i8[:, :, :], b1)",
+    # "b1[:], b1[:], b1[:], b1[:], i8[:], i8, i8, i8, f8[:, :, :], f8[:, :, :],""
+    # "f8[:, :, :], i8[:, :, :], i8[:, :, :], i8[:, :, :], b1)",
```

Comment:
Note that `ρL`/ `ρR` / `IL` / `IR` are 2D (see `_stump`). So, I think you should have written `f8[:, :]` for left/right pearson profiles and `i8[:, :]` for their corresponding indices.

Reply:
Fixed!